my_pyscf/mcscf/lasscf_rdm.py

```diff
@@ -179,8 +192,8 @@ def kernel (las, mo_coeff=None, casdm1frs=None, casdm2fr=None, conv_tol_grad=1e-
     converged = False
     t2 = (t1[0], t1[1])
     it = 0
-    for it in range (las.max_cycle_macro):
-        e_cas, casdm1frs, casdm2fr, rdmjk_conv = rdm_cycle (las, mo_coeff, casdm1frs,
+    for it in range (las.max_cycle_macro): ###las.rdm_cycle
+        e_cas, casdm1frs, casdm2fr, rdmjk_conv = las.rdm_cycle (mo_coeff, casdm1frs,
```

Comment:
This is kind of an example of a "hook" that my comment referred to above. `las.rdm_cycle` does one thing for lasscf_rdm.LASSCFNoSymm, and another thing for lasscf_rdm.extremeAsymLASSCF.